tests/pipeline/test_blaze.py

```diff
@@ -2179,19 +2179,31 @@ def test_checkpoints_out_of_bounds_macro(self):
         # provide two checkpoints, one before the data in the base table
         # and one after, these should not affect the value on the third
         asof_dates = pd.to_datetime(['2013-12-30', '2014-01-04'])
-        checkpoints = pd.DataFrame({
+        out_of_bounds = pd.DataFrame({
             'value': [-2, 2],
             'asof_date': asof_dates,
             'timestamp': asof_dates + pd.Timedelta(hours=23),
         })
 
-        self._test_checkpoints_macro(checkpoints)
+        # Add a single checkpoint on the query day with a timestamp of exactly
+        # the data query time. This should not get pulled to overwrite the
+        # expected data on the 3rd.
+        exact_query_time = pd.DataFrame({
+            'value': [1],
+            'asof_date': [pd.Timestamp('2014-01-02')],
+            'timestamp': [pd.Timestamp('2014-01-03')],
```

Comment:
It's not clear to me how these values are connected to the description in the comment above. What's enforcing that this is the leading data query cutoff time of the query we're going to make? Are these just the values that `test_checkpoints_*` uses by default? If so, it might help clarify to lift this value to a global constant that's shared with `test_checkpoints_*`